README.md

```diff
@@ -47,36 +46,33 @@ $ yarn db:start
 $ yarn dev
 ```
 
-By default, the app will run at: http://localhost:3000/
+- By default, the app will run at: http://localhost:3000/
 
-If `yarn db:start` failed and `localhost:5050` isn't working, a docker
-container, volume or image may be corrupted and need to be pruned.
+- If `yarn db:start` failed and `localhost:5050` isn't working, a docker
+  container, volume or image may be corrupted and need to be pruned.
 
-Build for production and start application:
+### Development
+
+- [Code Reviews](./docs/codeReview.md)
+- [Create new GraphQL Mutations](./scripts/codeshift/README.md)
+- [Docker](./docker/README.md)
+- [Dev.js](./scripts/README.md)
+- [File Storage (CDN, Local, S3)](./packages/server/fileStorage/README.md)
+- [GraphiQL, Private Schema Admin](./packages/server/graphql/intranetSchema/README.md)
+- [GraphQL Executor](./packages/gql-executor/README.md)
+- [Integrations (GitHub, Jira, Slack, etc.)](./docs/integrations.md)
+- [PostgreSQL](./packages/server/postgres/README.md)
+- [RethinkDB](./packages/server/db/README.md)
```

Comment:
-1 This link isn't working for me. Looks like it should be `- [RethinkDB](./packages/server/database/README.md)`